.github/workflows/publish.yaml

```diff
@@ -17,5 +17,6 @@ jobs:
   publish:
     if: ${{ github.repository_owner == 'google' }}
     uses: dart-lang/ecosystem/.github/workflows/publish.yaml@main
-    with:
-      write-comments: false
+    permissions:
```

Comment:
We want the publish and post_summaries workflows to look like those in https://github.com/dart-lang/core/tree/main/.github/workflows. Separating them out means that people who can't commit to the repo can still get publish feedback on their PRs.

The `write-comments: false` blurb here means that the associated post_summaries will pick up the content from the publish workflow and be the one to write it to the PR.

Reply:
I was getting failures on this.

I looked at `blast_repo` for the tweak. Maybe that should be updated?